### PR TITLE
MNINOR add common test elements to _openmetadata_testutils module

### PR DESCRIPTION
**Pull request by TeddyCr**
### Describe your changes:
- minor fix to allow explicit sharing of test bits across OM library. Fixes issues with dev environment not supporting runtime patching of tests bits

--
**Note:** this should ultimate be moved to its own separate package. 

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] My PR title is `Fixes <issue-number>: <short explanation>`
- [x] I have commented on my code, particularly in hard-to-understand areas. 
- [x] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

Improvement
- [x] I have added tests around the new logic.
- [x] For connector/ingestion changes: I updated the documentation.

### CODE REVIEW

File: ingestion/pyproject.toml
Comment (sushi30) [inline ingestion/pyproject.toml:1]: add the testutils package to `[tool.setuptools.packages.find]` under `exclude`.

**Review by sushi30 — CHANGES_REQUESTED** [review]

**Review by IceS2 — COMMENTED** [review]
Not sure I understand why it couldn't leave within the tests/ folder. Could you explain it so I understand what's going on?

Thanks!

File: ingestion/pyproject.toml
Comment (TeddyCr) [reply ingestion/pyproject.toml:1]: @sushi30 if we do that, then it is not importable.

Comment (sushi30) [reply ingestion/pyproject.toml:1]: then maybe we should put it outside the of the `metadata` dir and install it as part of dev dependencies.

**Review by sushi30 — APPROVED** [review]

**Pull request merged**